tests/stockfish/test_models.py

```diff
@@ -408,3 +408,26 @@ def test_make_moves_transposition_table_speed(self):
             total_time_calculating_second += default_timer() - start
 
         assert total_time_calculating_first < total_time_calculating_second
+
+    def test_benchmark(self, stockfish):
+        defaults = stockfish.benchmark()
+        assert defaults.split(" ")[0] == "Nodes/second"
+
+        valid_options = stockfish.benchmark(
+            ttSize=64,
+            threads=2,
+            limit=1000,
+            limitType="movetime",
+            evalType="classical",
+        )
+        assert valid_options.split(" ")[0] == "Nodes/second"
+
+        invalid_options = stockfish.benchmark(
+            ttSize=2049,
+            threads=0,
+            limit=0,
+            fenFile="./fakefile.fen",
+            limitType="fghthtr",
+            evalType="",
+        )
+        assert invalid_options.split(" ")[0] == "Nodes/second"
```

Comment:
What is the difference between the results in three different test cases?

`defaults`, `valid_options`, `invalid_options`? 

```assert invalid_options.split(" ")[0] == "Nodes/second"```

Reply:
There should be no difference. the benchmark returns a performance metric will be different every test. But these tests should ALWAYS finish, even when given invalid options. So as long as it runs the command and returns the last line of the output, it has worked.

Reply:
Ok

Reply:
Added comments to the tests